br/pkg/lightning/backend/local/local.go

```diff
@@ -782,7 +783,12 @@ func (local *local) WriteToTiKV(
 	size := int64(0)
 	totalCount := int64(0)
 	firstLoop := true
-	regionMaxSize := regionSplitSize * 4 / 3
+	// if region-split-size <= 96MiB, we bump the threshold a bit to avoid too many retry split
+	// because the range-properties is not 100% accurate
```

Comment:
Can we desrease `defaultPropSizeIndexDistance` to get more accurate results? Since it doesn't cost too much memory.

Reply:
I have tested smaller value 1MiB/10000, but there is no much difference. Maybe we can do deeper investigation to know the root reason.😅